src/MOI_wrapper.jl

```diff
@@ -2525,3 +2554,140 @@ function MOI.supports(
 )
     return true
 end
+
+###
+### VectorOfVariables-in-SecondOrderCone
+###
+
+function _info(
+    model::Optimizer,
+    c::MOI.ConstraintIndex{MOI.VectorOfVariables, MOI.SecondOrderCone}
+)
+    if haskey(model.quadratic_constraint_info, c.value)
+        return model.quadratic_constraint_info[c.value]
+    end
+    throw(MOI.InvalidIndex(c))
+end
+
+function MOI.add_constraint(
+    model::Optimizer, f::MOI.VectorOfVariables, s::MOI.SecondOrderCone
+)
+    if length(f.variables) != s.dimension
+        error("Dimension of $(s) does not match number of terms in $(f)")
+    end
+    # SOC is the cone: t ≥ ||x||₂ ≥ 0. In quadratic form, this is
+    # t² - Σᵢ xᵢ² ≥ 0 and t ≥ 0.
+
+    # First, check the lower bound on t.
+
+    # TODO(odow): a better way to set the lower bound.
+    t_info = _info(model, f.variables[1])
+    set_dblattrelement!(model.inner, "LB", t_info.column, 0.0)
```

Comment:
Any ideas for how to do this? Because what if `t` already has a lower bound? I almost want to add a dummy variable with `t = t_dummy`, but then I'd have to account for the un-used variable elsewhere in the wrapper, leading to more complexity.

Reply:
The MOI wrapper would know if there is an existing LB.
If there is one I would check what it is, if its less than zero, maybe, throw a warning?
If there is no LB we could add a the LB constraint OR only add this in the solver level and have an extra field in the optimizer to take car of this soc lb list of variables.

Reply:
```julia
if !MOI.is_valid(model, MOI.ConstraintIndex{MOI.SingleVariable, MOI.GreaterThan{Float64}(f.variables[1].value))
    # check that the bound is nonnegative, otherwise do
    MOI.add_constraint(model, MOI.SingleVariable(f.variables[1]), MOI.GreaterThan(0.0)) # This constraint should be hidden because it has not been added by the user. This is what `src/Bridges/bridge_optimizer.jl` does
end
```